set.mm

```diff
@@ -359917,157 +359973,252 @@ or are almost disjoint (the interiors are disjoint).  (Contributed by
     dvadd.y $e |- ( ph -> Y C_ S ) $.
     ${
       dvaddbr.s $e |- ( ph -> S C_ CC ) $.
-      dvadd.k $e |- ( ph -> K e. V ) $.
-      dvadd.l $e |- ( ph -> L e. V ) $.
       dvadd.bf $e |- ( ph -> C ( S _D F ) K ) $.
       dvadd.bg $e |- ( ph -> C ( S _D G ) L ) $.
       dvadd.j $e |- J = ( TopOpen ` CCfld ) $.
       $( The sum rule for derivatives at a point.  For the (simpler but more
          limited) function version, see ~ dvadd .  (Contributed by Mario
          Carneiro, 9-Aug-2014.)  (Revised by Mario Carneiro, 28-Dec-2016.) $)
       dvaddbr $p |- ( ph -> C ( S _D ( F oF + G ) ) ( K + L ) ) $=
```

Comment:
removing hypotheses counts as a revision and would get a revision tag

Reply:
Done